GAE/src/org/waterforpeople/mapping/app/web/RawDataRestServlet.java

```diff
@@ -415,7 +394,22 @@ private void updateDataPointLocation(SurveyedLocale dataPoint,
             }
         }
     }
-
+    
+    private void sendProcessingMessages(SurveyInstance domain) {
+        // send async request to summarize the instance
+        QueueFactory.getQueue("dataSummarization").add(
+                TaskOptions.Builder.withUrl("/app_worker/datasummarization")
```

Comment:
Lines 401-406, maybe the constants should be extracted? Some of them seem to be (see lines 407-409)